core/backend/src/test/standalone/RenderMaterialElement.test.ts

```diff
@@ -358,6 +358,13 @@ describe("RenderMaterialElement", () => {
         UnknownTexture: { TextureId: "CLONED" },
         NoTextureId: { OtherProp: 1 },
       });
+
+      jsonProps.materialAssets.renderMaterial.Map = {Pattern: undefined};
+      material.update();
+      // eslint-disable-next-line @typescript-eslint/dot-notation
+      RenderMaterialElement["onCloned"](context, sourceProps, targetProps);
+      // keep the sourceMap the same in targetProps
+      expect(targetProps.jsonProperties?.materialAssets?.renderMaterial?.Map?.Pattern).to.be.undefined;
```

Comment:
Couldn't this return undefined even if it didn't make it to Pattern?

I.e. if anything in the chain (JsonProperties, materialAssets, rendermaterial, map) is undefined it returns undefined.